src/sentry/static/sentry/app/components/events/interfaces/exceptionStacktraceContent.tsx

```diff
@@ -36,7 +36,7 @@ const ExceptionStacktraceContent = ({
 
   if (
     stackView === STACK_VIEW.APP &&
-    stacktrace.frames.filter(frame => frame.inApp).length === 0 &&
+    (stacktrace.frames ?? []).filter(frame => frame.inApp).length === 0 &&
```

Comment:
according to python code, this can also be undefined, so I thought it was better to update our types as well